docs/installation.md

```diff
@@ -0,0 +1,23 @@
+# Installation Guide
+
+## Requirement
+
+This SDK requires JDK 1.6+. We use [okhttp](https://github.com/square/okhttp) and [org.json](https://github.com/stleary/JSON-java) to manage project dependences.
+
+
+## Download
+
+Download [the latest JAR](https://github.com/yunify/qingstor-sdk-java/releases/download/2.1.1/qingstor-sdk-java-2.1.2.jar) or grab via Maven:
```

Comment:
Use https://github.com/yunify/qingstor-sdk-java/releases/latest will be better,  we don't need to maintain another version number in this project.